sup3r/pipeline/forward_pass.py

```diff
@@ -1587,16 +1589,20 @@ def _reshape_data_chunk(model, data_chunk, exo_data):
         i_lr_s : int
             Axis index for the low-resolution spatial_1 dimension
         """
-
+        current_model = None
```

Comment:
@grantbuster, what do you think about this addition? I added this so we could run with multi step spatial only models but I feel iffy about allowing len(exo_data) > len(model.models) in general.

Reply:
hmmm you might need to explain to me how this enables multi step spatial only. How many exo arrays for how many models? Honestly we should refactor this at some point anyway and make it more explicit what exo data is going to what model. too much implicit logic that we have to manage right now.

Reply:
With two spatial models there was an indexing error - its needs 3 exo arrays but only has two models. When this is a spatial model and a temporal model it just needs 2 exo arrays.

Reply:
Yeah i think this just points to the larger issue of implicit logic to handle these exo array mappings to models... Really we should make a dictionary of exo data in the config, e.g. {'exo_100km': {...}, 'exo_20km': {...}} and then have a mapping to models e.g. {'model0': [exo_100km, exo_20km], 'model3': [exo...]}.

If you have enough time to do this serious refactor i'd say go for it. If not, whatever logic fixes the problem is probably fine for now although temporary until we have the funds to do a better solution. 

Does that make sense? Thoughts?

Reply:
Yeah definitely makes sense. I can add the refactor to my list but lets implement a quicker fix for now.

Reply:
Agreed!!